src/QMCWaveFunctions/tests/test_RotatedSPOs_LCAO.cpp

```diff
@@ -361,4 +361,188 @@ TEST_CASE("Rotated LCAO WF with jastrow", "[qmcapp]")
   CHECK(dhpsioverpsi[1] == ValueApprox(10.047601212881027));
   CHECK(dhpsioverpsi[2] == ValueApprox(2.0820644399551895));
 }
+
+// Test the case where the rotation has already been applied to
+// the MO coefficients in the input file.
+// Should give the same results as the "Rotated LCAO WF zero angle" test case
```

Comment:
In the added two test cases, the wavefunction/orbitals are the same as "Rotated LCAO WF1" with (0.1, 0.2).
Since the derivatives are computed as the these of the current rotation, the reference numbers here are the same as "Rotated LCAO WF1". 
But why the comment says
"Should give the same results as the "Rotated LCAO WF zero angle" test case"

"Rotated LCAO WF zero angle" is different WF compared to "Rotated LCAO WF1" and the derivatives cannot be compared at a given electron configuration.

Or maybe that is just typo? Maybe we should rename "WF1" as "WF (0.1, 0.2)" to help reading.

Reply:
It's a typo. I'll either fix it to be consistent or find a better name (as you suggest).